src/components/header/header.css

```diff
@@ -0,0 +1,138 @@
+header{
+	background-color: #008AFF;
+}
+
+.header-content{
+	flex-wrap: wrap;
+	margin: 0 auto;
+	max-width: 1500px;
+	display: flex;
+	min-height: 100px;
+}
+
+.header-content--narrow{
+  max-width: 1000px;
+}
+
+.header-content__button-container, .header-content__title-container, .header-content__tools-container{
+	display: flex;
+	flex: 1;
+}
+
+.header-content__button-container{
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+}
+
+.header-content__switch-mode-button{
+  text-align: center;
+	border:  2px solid #008AFF;
+	border-radius: 2px;
+	color: white;
+	cursor: pointer;
+	font: 400 24px 'Roboto', sans-serif;
```

Comment:
Login pull request #6 is adding Tajawal font. Please decide with the team members which one you will use.